### PR TITLE
Updated Company Document for git client

### PR DESCRIPTION
**Pull request by swapnilBhi**

### CODE REVIEW

File: sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs [hunk 1]
```diff
@@ -116,24 +116,26 @@ private static async Task<string> Run(AppConfiguration configuration)
...
                 cvrs: cvrs,
                 id: Guid.NewGuid(),
-                cpr: configuration.Cpr,
                 companyDocumentType: configuration.CompanyDocumentType,
                 documentUrl: "https://citizen-documents.prod.kmdlogic.io",
                 retentionPeriodInDays: configuration.RetentionPeriodInDays,
                 status: "Completed",
                 fileName: "test.png",
-                documentName: "test.png");
+                documentName: "test.png",
```
Comment (deviprasadkotian) [inline sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs]: Please remove the file extensions.

**Review by deviprasadkotian — DISMISSED** [review]

File: sample/Kmd.Logic.CitizenDocuments.Client.Sample/AppConfiguration.cs [hunk 2]
```diff
@@ -28,5 +28,10 @@ internal class AppConfiguration
         public string SendDocumentType { get; set; } = "alm brev";
 
         public string Title { get; set; } = "test";
+
+        public string Sender { get; set; }
```
Comment (paimaam) [inline sample/Kmd.Logic.CitizenDocuments.Client.Sample/AppConfiguration.cs]: Assign some dummy values to this

**Review by paimaam — DISMISSED** [review]
LGTM

**Review by Araballi — APPROVED** [review]
LGTM

**Pull request merged**